schema/events.graphql

```diff
@@ -18,6 +18,17 @@ type Event @entity @schema(name: "admin") {
   data: EventData!
 }
 
+type FutureNotificationOrionEvent @entity {
+  "notification type that will be created"
+  notificationType: NotificationType!
+
+  "block after which notification should be created"
+  executionBlock: Int!
+
+  "related event for on chain notifications"
+  event: Event
+}
```

Comment:
Possibly, I don't really remember the logic after that long

Reply:
But I assume the main goal was to transfer `dispatchBlock`, so if it can be done easier way, then sure. Do we have `Event` field in the `NotificationType`? I think the notification builder requires it.